addons/core/functions/fnc_ClientInit.sqf

```diff
@@ -172,35 +173,24 @@ if (player call TFAR_fnc_isForcedCurator) then {
 
     if !(TFAR_currentUnit getVariable ["TFAR_HandlersSet",false]) then {
         TFAR_currentUnit addEventHandler ["Take", {
-            private _class = configFile >> "CfgWeapons" >> (_this select 2);
-            if !(isClass _class) exitWith {};
-            if (isNumber (_class >> "tf_radio")) exitWith {
-                [(_this select 2), getPlayerUID player] call TFAR_fnc_setRadioOwner;
-            };
-            if (isNumber (_class >> "tf_prototype")) then {
-                call TFAR_fnc_radioReplaceProcess;
+            params ["_unit", "", "_item"];
+            if (_item call TFAR_fnc_isPrototypeRadio) then {
+                private _classes = _unit call TFAR_fnc_getDefaultRadioClasses;                
+                [_unit, _classes select 2] remoteExec ["TFAR_fnc_replaceSwRadiosServer", 2];
```

Comment:
This is invalid. fnc_replaceSwRadiosServer only takes one parameter.

Reply:
Nope, there are two parameters. Need to update TFAR_fnc_replaceSwRadiosServer header.